src/plugins/score-plugin-scenario/Scenario/Document/Tempo/TempoView.hpp

```diff
@@ -56,8 +57,51 @@ class TempoPresenter final
       QObject* parent)
       : CurveProcessPresenter{style, layer, view, context, parent}
   {
+    // only disable the curve when using the first 2 inlets
+    for (int i = 0; i < 2; i++)
+    {
+      QObject::connect(
```

Comment:
so, here I'd recommend just having a lambda instead of the two functions: 


      QObject::connect(layer.inlets()[i], &Process::Inlet::cablesChanged,
                                      this, [] { disableIfNeeded(); }); 
                                      
it should work for both cases and take less space.

Reply:
Yes of course !
Although I require this in the capture ie:
```
          [this] { disableIfNeeded(); });

```